src/views/virtualmachines/actions/components/CloneVMModal/CloneVMModal.tsx

```diff
@@ -57,7 +57,8 @@ const CloneVMModal: React.FC<CloneVMModalProps> = ({ vm, isOpen, onClose }) => {
   const { projects, pvcs, loaded } = useCloneVMResources(vm);
 
   const projectNames = React.useMemo(
-    () => (projects || [])?.map((project) => project.metadata.name),
+    () =>
+      (projects || [])?.map((project) => project.metadata.name).sort((a, b) => a.localeCompare(b)),
```

Comment:
Please consider moving this sorting inside the hook so if used again, we would only need to sort once :) 
also consider in the hook assign projects with [] as default value, and import getName util function, than the sort would look like this: 
`projects.map(getName).sort((a, b) => a.localeCompare(b))`